media/css/security/security.scss

```diff
@@ -2,16 +2,12 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
-$font-path: '/media/protocol/fonts';
-$image-path: '/media/protocol/img';
-
-@import '~@mozilla-protocol/core/protocol/css/includes/lib';
-@import 'components/advisory',
-    'components/data-table',
-    'components/level',
-    'components/links',
-    'components/note';
+@use 'components/advisory';
+@use 'components/data-table';
+@use 'components/level';
+@use 'components/links';
+@use 'components/note';
 
 .pgp-key {
-    overflow-x: scroll;
+    overflow-x: auto; // only add scrollbar with horizontal overflow
```

Comment:
prod: sometimes the whole block is in view but there's still an empty scrollbar
<img width="1126" alt="Screenshot 2025-01-13 at 3 29 53 PM" src="https://github.com/user-attachments/assets/49eb6471-e95b-4d5d-8c4d-8b3c3280065a" />

PR: keeps the scrollbar only when overflow happens
<img width="516" alt="Screenshot 2025-01-13 at 3 30 20 PM" src="https://github.com/user-attachments/assets/7ee7d160-518f-4da6-8063-86205be9a8a8" />